service/core.go

```diff
@@ -226,17 +227,17 @@ func (c *Core) requestLogger(r *http.Request) *zap.Logger {
 	return c.logger.With(zap.String("request_id", api.RequestIDFromContext(r.Context())))
 }
 
-func (c *Core) publishEvent(event string, data interface{}) {
+func (c *Core) publishEvent(w *ResponseWriter, name string, data interface{}) {
+	zv, err := zson.MarshalZNG(data)
+	if err != nil {
+		w.Logger.Error("error marshaling published event", zap.Error(err))
```

Comment:
Capitalize the first word of a log string.
```suggestion
		w.Logger.Error("Error marshaling published event", zap.Error(err))
```